modules/standard/Path.chpl

```diff
@@ -433,6 +434,15 @@ proc isAbsPath(name: string): bool {
   }
 }
 
+private proc joinPathComponent(comp: string, ref result: string) {
```

Comment:
Let's add a doc string here. Developers matter too!

Reply:
When you say to add a doc string, do you mean just add a comment describing what the function does? Or a full on piece of documentation?

Reply:
just a simple one-liner: `/* ... */`